liquidctl/driver/gigabyte_rgb_fusion.py

```diff
@@ -0,0 +1,328 @@
+        'flash':                            (0x03, 0x00, 0x01, 0x01, 0x0f, 0x64, 1, 1),
+        'double-flash':                     (0x03, 0x00, 0x01, 0x02, 0x0f, 0x64, 1, 1),
+        'color-cycle':                      (0x04, 0x07, 0x00, 0x00, 0x0f, 0x64, 0, 0),
+    }
+    
+    _PULSE_SPEEDS = {
+        'slowest':                          (0x40, 0x06, 0x40, 0x06, 0x20, 0x03),
+        'slower':                           (0x78, 0x05, 0x78, 0x05, 0xbc, 0x02),
+        'normal':                           (0xb0, 0x04, 0xb0, 0x04, 0xf4, 0x01),
+        'faster':                           (0xe8, 0x03, 0xe8, 0x03, 0xf4, 0x01),
+        'fastest':                          (0x84, 0x03, 0x84, 0x03, 0xc2, 0x01),
+        'ludicrous':                        (0x20, 0x03, 0x20, 0x03, 0x90, 0x01),
+    }
+    
+    _FLASH_SPEEDS = {
+        'slowest':                          (0x64, 0x00, 0x64, 0x00, 0x60, 0x09),
+        'slower':                           (0x64, 0x00, 0x64, 0x00, 0x90, 0x08),
+        'normal':                           (0x64, 0x00, 0x64, 0x00, 0xd0, 0x07),
+        'faster':                           (0x64, 0x00, 0x64, 0x00, 0x08, 0x07),
+        'fastest':                          (0x64, 0x00, 0x64, 0x00, 0x40, 0x06),
+        'ludicrous':                        (0x64, 0x00, 0x64, 0x00, 0x78, 0x05),
+    }
+
+    _DOUBLE_FLASH_SPEEDS = {
+        'slowest':                          (0x64, 0x00, 0x64, 0x00, 0x28, 0x0a),
+        'slower ':                          (0x64, 0x00, 0x64, 0x00, 0x60, 0x09),
+        'normal':                           (0x64, 0x00, 0x64, 0x00, 0x90, 0x08),
+        'faster':                           (0x64, 0x00, 0x64, 0x00, 0xd0, 0x07),
+        'fastest':                          (0x64, 0x00, 0x64, 0x00, 0x08, 0x07),
+        'ludicrous':                        (0x64, 0x00, 0x64, 0x00, 0x40, 0x06),
+    }
+
+    _COLOR_CYCLE_SPEEDS = {
+        'slowest':                          (0x78, 0x05, 0xb0, 0x04, 0x00, 0x00),
+        'slower':                           (0x7e, 0x04, 0x1a, 0x04, 0x00, 0x00),
+        'normal':                           (0x52, 0x03, 0xee, 0x02, 0x00, 0x00),
+        'faster':                           (0xf8, 0x02, 0x94, 0x02, 0x00, 0x00),
+        'fastest':                          (0x26, 0x02, 0xc2, 0x01, 0x00, 0x00),
+        'ludicrous':                        (0xcc, 0x01, 0x68, 0x01, 0x00, 0x00),
+    }
+    
+    _RGB_FUSION_SPEEDS = {
+        'pulse':                            (_PULSE_SPEEDS),
+        'flash':                            (_FLASH_SPEEDS),
+        'double-flash':                     (_DOUBLE_FLASH_SPEEDS),
+        'color-cycle':                      (_COLOR_CYCLE_SPEEDS),    
+    }
+
+    def __init__(self, device, description, speed_channel_count, color_channel_count, **kwargs):
+        """Instantiate a driver with a device handle."""
+        speed_channels = {'fan{}'.format(i + 1): (1 << i)
```

Comment:
You can remove the `speed_channels` field and the corresponding `speed_channel_count` if you're not using them.

Reply:
Will do.

Reply:
Not really a review, just noticed that `speed_channel_count` should also be removed from this constructor's argument list, as well as from `SUPPORTED_DEVICES`.